### PR TITLE
Code climate setup

### PR DESCRIPTION
**Pull request by emduc**
Final setup of code climate according to the [Operations' guide](https://github.com/sweng-epfl/public/blob/main/sdp/guides/Operations.md)

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ check_android_task:
     API_LEVEL: 30
     TARGET: google_apis
     ARCH: x86
+    CC_TEST_REPORTER_ID: ENCRYPTED[!2d90341ad69e7db861f6ea73103aa743c26ba4de3c4aabb5083ea778ca7d68fbef69bc9762a46a4825f11abd214e3495!]
```
Comment (emduc) [inline .cirrus.yml:7]:
```suggestion
    CC_TEST_REPORTER_ID: ENCRYPTED[fc8a7cbb52ff46fb28547bca7cfba41d1e380f2912b9672e6d41b19dcb414afe43c14362121c6299e834e7d8b5364b18]
```

**Review by Adrinels — APPROVED** [review]
As expected in the bootcamp. Nice!

**Comment by wiam8** [comment]
Perfect work!

**Review by wiam8 — APPROVED** [review]
Perfect work!

**Pull request merged**